src/components/TaskCard/AddTaskCard.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import PropTypes from "prop-types";
+import { sanitizeText } from "utils/helpers/sanitizeText.js";
+import { addTask } from "store/actions/";
+import { KEY_ENTER } from "utils/constant/form";
+import "components/TaskCard/index.scss";
+
+const AddTaskCard = ({ onCreateCard }) => {
```

Comment:
does the application creates a Card or a Task?

Reply:
renamed onCreateCard to onCreateTask. Thank you.